pkg/products/amqstreams/reconciler.go

```diff
@@ -215,11 +215,11 @@ func (r *Reconciler) handleProgressPhase(ctx context.Context, client k8sclient.C
 	}
 
 	//expecting 8 pods in total
-	if len(pods.Items) < 8 {
+	if len(pods.Items) == 8 {
```

Comment:
This is a small thing. Can the value of 8 be turned into a variable instead? A well named variable would make the comment above not required.

Reply:
Should this if block be removed if there is no need to check the pod count? :thinking: 

Depending on the decision, some unit tests would need to be updated as the expected phase returned is now different in these tests as they create pods in the ready or unready state:
* https://github.com/integr8ly/integreatly-operator/blob/ee67540e2bf7b580366731804f8650f8dcce2442/pkg/products/amqstreams/reconciler_test.go#L365

* https://github.com/integr8ly/integreatly-operator/blob/ee67540e2bf7b580366731804f8650f8dcce2442/pkg/products/amqstreams/reconciler_test.go#L236